java/examples/datasets/JavaDatasetExample.sh.in

```diff
@@ -103,17 +104,21 @@ COPY_LIBFILES_TO_BLDLIBDIR()
             INODE_SDIR=`$LS -i -d $SDIR | $AWK -F' ' '{print $1}'`
             INODE_DDIR=`$LS -i -d $BLDLIBDIR | $AWK -F' ' '{print $1}'`
             if [ "$INODE_SDIR" != "$INODE_DDIR" ]; then
-                $CP -f $tstfile $BLDLIBDIR
```

Comment:
The 'R' option preserves soft links correctly, instead of letting them be duplicated into new copies of the linked file.